wscript

```diff
@@ -136,7 +136,7 @@ def build(bld):
     if 'woss-ns3' in bld.env['MODULES_NOT_BUILT']:
         return
 
-    module = bld.create_ns3_module('woss-ns3', ['network', 'energy', 'mobility', 'uan'])
+    module = bld.create_ns3_module('woss-ns3', ['netanim','network', 'energy', 'mobility', 'uan'])
```

Comment:
please add a space after the comma between netanim and network

`    module = bld.create_ns3_module('woss-ns3', ['netanim', 'network', 'energy', 'mobility', 'uan'])`